README.md

```diff
@@ -15,20 +15,56 @@
 |:----------------:|:----------------------------:|
 | v0.1.*           | v1.5.*                       |
 
-## Installation
+## Using this starter kit
 
-Add the following dependency to your Akeneo PIM project with this shell command:
+### Context
+To write an extension, you will need to integrate your extension in an existing PIM installation.
+We will use the PIM Community Standard Edition as the host of our connector.
 
+### Initialisation
+Create a new host project with the Standard Edition
+
+```
+composer create-project --prefer-dist akeneo/pim-community-standard /home/myhome/pim-project "1.5.*@stable"
```

Comment:
`/path/to/project`. Otherwise you get a permission error if copy and paste (and let's assume some will copy/paste) this command line.